src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -396,6 +396,14 @@ class DistributedClosestPointImpl
     m_sqDistanceThreshold = sqThreshold;
   }
 
+  /// Sets the allocator ID to the default associated with the execution policy
+  void setAllocatorID(int allocatorID)
+  {
+    SLIC_ASSERT(allocatorID != axom::INVALID_ALLOCATOR_ID);
```

Comment:
Thought: Should we allow the user to set it to `axom::INVALID_ALLOCATOR_ID`? e.g. to clear it to the default?

Reply:
Interesting idea.  How about if we make `setDefaultAllocatorID` public, to do the same thing explicitly?